web-registry/src/pages/CreditClassDetails/CreditClassDetails.AdditionalInfo.tsx

```diff
@@ -10,15 +13,21 @@ import {
 
 import { MetaDetail } from 'components/molecules';
 
+import {
+  CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_CLASS_ID_TOOLTIP,
+  CREDIT_CLASS_DETAILS_ADDITIONAL_INFO_HELPER_TEXT,
+} from './CreditClassDetails.constants';
 import { ApprovedMethodologiesList } from './CreditClassDetails.ApprovedMethodologies';
 import { BufferPoolAccounts } from './CreditClassDetails.BufferPoolAccounts';
 
 interface AdditionalInfoProps<T extends CreditClassMetadataLD> {
+  classId?: string;
   metadata?: T;
   creditTypeName?: string;
 }
 
 const AdditionalInfo = <T extends CreditClassMetadataLD>({
+  classId,
```

Comment:
on line 34, we just return null if there's no metadata but since we always display the credit class id, we should probably remove this check

Reply:
Removed https://github.com/regen-network/regen-web/pull/1985/commits/ec8987462fb24e4130516256bbbcfe84bae032de